types/extract-files/index.d.ts

```diff
@@ -1,20 +1,32 @@
-// Type definitions for extract-files 3.1
+// Type definitions for extract-files 8.1
 // Project: https://github.com/jaydenseric/extract-files#readme
 // Definitions by: Edward Sammut Alessi <https://github.com/Slessi>
+//                 Alex K <https://github.com/lynxtaa>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+export type ExtractableFileMatcher = (value: any) => boolean;
```

Comment:
@lynxtaa Use a type guard

```ts
export type ExtractableFile = File | Blob | ReactNativeFile;
export type ExtractableFileMatcher = (value: any) => value is ExtractableFile;
```

Reply:
Done. Although I've decided not export `ExtractableFileMatcher` because custom matchers shouldn't use this type (see updated test in `extract-files-tests.ts`)